applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp

```diff
@@ -223,12 +276,12 @@ inline void InterfacePreprocessCondition::GenerateEdgeCondition(
                 // We initialize a vector for the IDs
                 std::vector<std::size_t> condition_ids(2);
 
-                Line2D2< Node<3> > lin_1(EdgeGeometry(0), EdgeGeometry(1));
+                Line2D2< NodeType > lin_1(EdgeGeometry(0), EdgeGeometry(1));
```

Comment:
I think the edge is a Line2D3, Can't you use it as such?

Reply:
This is for 2D, maybe edge is not the correct word